x/migration/module/cmd/claim_morse_supplier_bulk.go

```diff
@@ -335,6 +343,18 @@ func runClaimSuppliers(cmd *cobra.Command, _ []string) error {
 			shannonOwnerAddress = ownerAddressToMClaimableAccountMap[morseOutputAddress].ShannonDestAddress
 		}
 
+		// TODO(@jorgecuesta): I don't like this,
```

Comment:
Why not just change `loadTemplateSupplierStakeConfigYAML` to return a value instead of a pointer?

Reply:
It's only used in one place. Let's do it.

![Screenshot 2025-06-06 at 10 29 28 AM](https://github.com/user-attachments/assets/5dd2665a-2028-4d7e-b673-4590b786c575)

Reply:
> It's only used in one place. Let's do it.
@olshansky this use the shared struct which is what I dont want to modify unless you explicitly require that, the function you search is the I defined on the other CLI because the common one is validating file by default fail because owner and operator are not set.

Reply:
I figure it out a way to use the template without call the read file many times.